pyproject.toml

```diff
@@ -37,7 +37,7 @@ dependencies = [
     "jsonschema",
     "psutil",
     "pyedb>=0.4.0; python_version == '3.7'",
-    "pyedb>=0.24.0; python_version > '3.7'",
+    "pyedb>=0.27.0; python_version > '3.7'",
     "pyedb!=0.28.0; python_version > '3.7'",
```

Comment:
why we skip 0.28.0?

Reply:
pyaedt doesn't pass unittest with pyedb 0.28.0.

Reply:
We had an issue with pyedb `0.28.0` which broke the tests in Linux. This has been a work in progress and should be fixed in `0.29.0`. It was working with pyedb `0.29.dev0` a few days ago but failed when we tried to release `0.29.0` ....